client/init.c

```diff
@@ -29,6 +29,8 @@
 char *g_coinname, *g_progname;
 #define coinname   g_coinname
 
+#define ARG_EQUAL(a,b,c) strcmp(c, "")==0?strcmp(a, b) == 0:(strcmp(a, b)==0 || strcmp(a, c) == 0)
```

Comment:
The only comment - somewhere there are spaces, somewhere there are missed spaces. Can be fixed later, so I will merge it.